docs/reference/modules/uploadfs.md

```diff
@@ -181,6 +182,9 @@ The `key` option is used to pass the value of the `accessKeyId` to the `AWS.Cred
 ### `token`
 The `token` option is used to pass the value of the optional `sessionToken` to the `AWS.Credentials()` credentials object. See the [official documentation](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#constructor_details) for alternative ways to present the credentials.
 
+### `bucketObjectsACL`
+The `bucketObjectsACL` option sets the access control level for files uploaded to AWS S3, with a default of `public-read`. Setting this option to `private` ensures that only authorized users can access the files, enhancing security for sensitive content. This is useful when integrating with a CDN, where you can use signed URLs to securely serve private content to users.
```

Comment:
This will need work to address the use case: making the bucket private while keeping the files publicly available via CloudFront. As you note it doesn't seem useful to combine this technique with other S3 providers, but it is Amazon's recommended strategy to make the bucket contents private and then allow CloudFront to serve them publicly.

Reply:
I apologize for not making that clear the first time.

Reply:
Changed